components/script/build.rs

```diff
@@ -3,10 +3,45 @@
  * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
 
 use std::env;
+use std::path::{Path, PathBuf};
 
 fn main() {
-    println!(
-        "cargo:rustc-env=BINDINGS_OUT_DIR={}",
-        env::var("DEP_SCRIPT_BINDINGS_CRATE_OUT_DIR").unwrap(),
-    );
+    // copy include! files from script_bindings's OUT_DIR, to script's OUT_DIR
```

Comment:
As written, I am concerned that this will increase the build time of incremental builds where there are no changes to the generated code. Can we perform an mtime check and only copy files that are newer than the file in the destination?

Reply:
IIRC, maybe it would be enough to set that build.rs script gets rerun if changes only on specific files are detected. I will investigate more tomorrow.

Reply:
They key part why https://github.com/servo/servo/pull/36384/commits/4ef86a46b3ac0aa02ea34f1747bdbb91fc3ac732 works is [this](https://doc.rust-lang.org/cargo/reference/build-scripts.html#change-detection):
> If the build script inherently does not need to re-run under any circumstance, then emitting cargo::rerun-if-changed=build.rs is a simple way to prevent it from being re-run (otherwise, the default if no rerun-if instructions are emitted is to scan the entire package directory for changes).